<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UGENE Crash Reporter</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../crash_icon.png</normaloff>../crash_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>A fatal error has occurred during the program execution.
Would you like to send the following crash report to UGENE team?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="errorEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="emailLabel">
     <property name="text">
      <string>Please, enter your e-mail to stay in touch and help us improve UGENE (optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="emailLineEdit">
     <property name="maxLength">
      <number>200</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="additionalInfoLabel">
     <property name="text">
      <string>Additional information (optional)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="additionalInfoTextEdit"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Don't send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
